corehq/apps/domain/templates/domain/internal_calculations.html

```diff
@@ -39,7 +39,7 @@
     </script>
 {% endblock %}
 
-{% block main_column %}
+{% block page_content %}
     <div class="form form-horizontal">
```

Comment:
And if possible, please try to convert this form to a crispy form. Follow the guide here in the Django Form Checklist which shows you how to take an existing Django form and turn it into a Crispy Form. https://staging.commcarehq.org/styleguide/forms/#b3-migration